### DISCUSSION TITLE
Setting up autoformatting

### DISCUSSION THREAD

**Discussion by akupar** [widelands/widelands#5445] [General]
I have tried to set up autoformatting according to the instructions [here](https://github.com/widelands/widelands#code) for 3 or 4 times now, but I can't get it to work. Any ideas why?

**Comment by matthiakl** [comment]
Did you check, that actions are enabled in your fork generally? I don't know what the default value is. If actions are disabled initially, we should add a corresponding note to the readme.

![Bildschirmfoto vom 2022-06-06 20-45-04](https://user-images.githubusercontent.com/23122583/172226303-ec1ab810-5413-4b06-ac89-6d8f21235157.png)

**Reply by akupar** [reply]
Yes, that is enabled
